doc/TockBinaryFormat.md

```diff
@@ -155,6 +156,14 @@ struct TbfHeaderV2Permissions {
     length: u16,
     perms: [TbfHeaderDriverPermission],
 }
+
+// Kernel Version
+struct TbfHeaderV2KernelVersion {
+    base: TbfHeaderTlv,
+    length: u16,
+    major: u8,
+    minor: u8
```

Comment:
These are `u8`s here but seem to be `u16`s in the tock-tbf code?

Reply:
Fixed, sorry for that